### PR TITLE
New Korean translation

### PR DESCRIPTION
**Pull request by hyeyoom**
1. README-ko.md
2. Korean translation link(README-ko.md) into README.md

It is the best for digital video newbie!
I am so glad to translate this article into Korean first time.

### CODE REVIEW

**Comment by leandromoreira** [comment]
Thanks a lot @hyeyoom =)

Is there anybody else who knows korean and could code review this one?

**Comment by hyeyoom** [comment]
Oops My mistake. Sorry. I worked with origin file for the first time. I will fix the index in a few day. And i hope someone review my pr!

**Comment by silenc3502** [comment]
Hello, that translation has no major problems to understand contents. It's OK for the problem of typing that did not remove some English texts.

**Comment by leandromoreira** [comment]
> Oops My mistake. Sorry. I worked with origin file for the first time. I will fix the index in a few day. And i hope someone review my pr!

can you fix that ? ( I mean, the original index shouldn't be changed)

**Comment by hyeyoom** [comment]
Of course I can. I will fix that tonight!

File: README-ko.md [hunk 1]
```diff
@@ -271,45 +271,42 @@ G = Y - 0.344Cb - 0.714Cr
...
-An I-frame (reference, keyframe, intra) is a **self-contained frame**. It doesn't rely on anything to be rendered, an I-frame looks similar to a static photo. The first frame is usually an I-frame but we'll see I-frames inserted regularly among other types of frames.
-
 `I-frame(레퍼런스, 키프레임, 인트라)`은 **자기 자신을 포함한 프레임**입니다. 렌더링 하기 위해 어떤 것에도 의존적이지 않지요. I-frame은 마치 정적인 사진같습니다. 첫 번째 프레임은 보통 I-frame입니다. 하지만 I-frame은 일반적으로 다른 유형의 프레임의 사이사이에 삽입된다는 사실을 볼 수 있을 겁니다.  
 
 ![ball 1](/i/smw_background_ball_1.png "ball 1")
 
 ### P Frame (predicted)
 
-A P-frame takes advantage of the fact that almost always the current picture can be **rendered using the previous frame.** For instance, in the second frame, the only change was the ball that moved forward. We can **rebuild frame 1, only using the difference and referencing to the previous frame**.
+`P-frame`은 거의 항상 **이전의 프레임을 이용하여 렌더링 될 수 있다는 점**을 이점으로 취할 수 있습니다. 예를ㄷ를어 두 번째 프레임에서 변화라곤 공이 앞으로 움직인 것 뿐입니다. **변경된 부분을 사용해서 이 전의 프레임을 참고하여 프레임 1번을 다시 만들 수 있지요.**
```
[the remark this answers is not in the document]
Comment (leandromoreira) [reply README-ko.md]: yes, it can.

File: README-ko.md [hunk 2]
```diff
@@ -271,45 +271,42 @@ G = Y - 0.344Cb - 0.714Cr
 
 ### I Frame (인트라, 키프레임)
 
-An I-frame (reference, keyframe, intra) is a **self-contained frame**. It doesn't rely on anything to be rendered, an I-frame looks similar to a static photo. The first frame is usually an I-frame but we'll see I-frames inserted regularly among other types of frames.
-
 `I-frame(레퍼런스, 키프레임, 인트라)`은 **자기 자신을 포함한 프레임**입니다. 렌더링 하기 위해 어떤 것에도 의존적이지 않지요. I-frame은 마치 정적인 사진같습니다. 첫 번째 프레임은 보통 I-frame입니다. 하지만 I-frame은 일반적으로 다른 유형의 프레임의 사이사이에 삽입된다는 사실을 볼 수 있을 겁니다.
```
[the remark this answers is not in the document]
Comment (hyeyoom) [reply README-ko.md]: 오 감사합니다. 반영할게요.

File: README-ko.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (hyeyoom) [reply README-ko.md]: 오.. 지적 감사합니다.

File: README-ko.md [hunk 3]
```diff
@@ -757,34 +756,33 @@ In real life production systems, people often use both techniques to provide aut
...
 
-The content C1 is encrypted with a **symmetric-key K1** from the system DRM1, generating the **encrypted content C'1**.
+컨텐츠 C1은 시스템 DRM1의 **대칭키 K1**으로 암호화되어 **암호화된 컨텐츠 C'1**을 생성합니다.  
 
 ![drm general flow](/i/drm_general_flow.jpeg "drm general flow")
 
-The player P1, of a device D1, has two keys (asymmetric), a **private key PRK1** (this key is protected<sup>1</sup> and only known by **D1**) and a **public key PUK1**.
-
-> **<sup>1</sup>protected**: this protection can be **via hardware**, for instance, this key can be stored inside a special (read-only) chip that works like [a black-box](https://en.wikipedia.org/wiki/Black_box) to provide decryption, or **by software** (less safe), the DRM system provides means to know which type of protection a given device has.
+디바이스 D1의 플레이어 P1은 **비밀키 PRK1**(이 키는 보호되어 있으며<sup>1</sup> **D1**만 알고 있음)와 **공개키 PUK1**라는 두 개의 키(비대칭)를 가지고 있습니다.
```
[the remark this answers is not in the document]
Comment (hyeyoom) [reply README-ko.md]: 위키페디아에서 비밀 키로 용어를 통일하길래 이렇게 적어두었는데 확실히 구글 검색 결과 수는 개인키가 더 많군요. 수정하겠습니다.

File: README-ko.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (hyeyoom) [reply README-ko.md]: here 말씀하신거죠? ._.

**Comment by hyeyoom** [comment]
@yuntaek 파일이 수정되었습니다. 감사합니다! :) 💯 👍 
@leandromoreira I fixed all files including yuntakek's requests. Thank you.

File: README-ko.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (yuntaek) [reply README-ko.md]: here 에 제가 찾은 self-contained의 의미를 링크 걸어 놓은 거예요. 
반영 감사합니다.

**Pull request merged**